easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-gfbf-2022b.eb

```diff
@@ -0,0 +1,44 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+# Updated to foss 2021b: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'MoviePy'
+version = '1.0.3'
+
+homepage = "https://zulko.github.io/moviepy/"
+description = """MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,
+ title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects."""
+
+toolchain = {'name': 'gfbf', 'version': '2022b'}
+
+dependencies = [
+    ('FFmpeg', '5.1.2'),
+    ('Python', '3.10.8'),
+    ('Pillow', '9.4.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('tqdm', '4.64.1'),
+    ('imageio', '2.28.0'),
```

Comment:
We already have `imageio-2.31.1-foss-2022b.eb`, can't that be used as a dependency here?

Reply:
@boegel I no longer have access to a `foss/2022b` software stack but fine to change (and test with gh actions).